opentelemetry-appender-log/examples/logs-basic-in-memory.rs

```diff
@@ -25,7 +25,10 @@ async fn main() {
     log::set_max_level(Level::Info.to_level_filter());
 
     // Emit logs using macros from the log crate.
-    error!("hello from {}. My price is {}", "apple", 2.99);
+    let user = "apple";
+    let price = 2.99;
+
+    error!(user, price; "hello from {user}. My price is {price}");
```

Comment:
Do you think it is possible to add a serde specific example under feature flag ?